packages/twenty-server/src/engine/core-modules/user/user-vars/services/user-vars.service.ts

```diff
@@ -75,32 +76,32 @@ export class UserVarsService<
     ]).get(key) as KeyValueTypesMap[K];
   }
 
-  public async getAll({
-    userId,
-    workspaceId,
-  }: {
-    userId?: string;
-    workspaceId?: string;
-  }): Promise<Map<Extract<keyof KeyValueTypesMap, string>, any>> {
-    const userVarsWorkspaceLevel = await this.keyValuePairService.get({
+  public async getAll(
+    user: User,
+  ): Promise<Map<Extract<keyof KeyValueTypesMap, string>, any>> {
+    let result = await this.keyValuePairService.get({
       type: KeyValuePairType.USER_VAR,
-      userId: null,
-      workspaceId,
+      userId: user.id,
+      workspaceId: null,
     });
 
-    let userVarsUserLevel: any[] = [];
-
-    if (userId) {
-      userVarsUserLevel = await this.keyValuePairService.get({
-        type: KeyValuePairType.USER_VAR,
-        userId,
-      });
+    if (user.defaultWorkspaceId) {
```

Comment:
revert and only add workspaceId: null to compute userVarsUserLevel

Reply:
we will miss userId non null, and workspaceId non null variables